### PR TITLE
fix ineffectual assignment to err

### PR DESCRIPTION
**Pull request by zgfh**
### 1. Why is this pull request needed and what does it do?
fix ineffectual assignment to err
### 2. Which issues (if any) are related?
no
### 3. Which documentation changes (if any) need to be made?
no
### 4. Does this introduce a backward incompatible change or deprecation?
no

### CODE REVIEW

File: plugin/kubernetes/controller_test.go [hunk 1]
```diff
@@ -79,6 +79,9 @@ func generateEndpoints(cidr string, client kubernetes.Interface) {
 		}
 		ep.ObjectMeta.Name = "svc" + strconv.Itoa(count)
 		_, err = client.CoreV1().Endpoints("testns").Create(ep)
+		if err != nil {
```
Comment (miekg) [inline plugin/kubernetes/controller_test.go]: this should return the err; but this being test code I don't think we care **that** much

Comment (miekg) [reply plugin/kubernetes/controller_test.go]: idem for the other test functions

File: owners_generate.go [hunk 2]
```diff
@@ -26,7 +26,7 @@ func main() {
 	}
 
 	// each of the plugins, in case someone is not in the top-level one
-	err = filepath.Walk("plugin",
+	_ = filepath.Walk("plugin",
```
Comment (miekg) [inline owners_generate.go]: keep the `err` and `log.Fatal(err)` afterwards

Comment (zgfh) [reply owners_generate.go]: updated

**Comment by miekg** [comment]
the change in `owners_generate.go` is the only one I want to keep; there rest can be reverted.

**Comment by zgfh** [comment]
@miekg updated

**Pull request merged**